app/components/primer/beta/auto_complete/item.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render Primer::BaseComponent.new(**@system_arguments) do %>
```

Comment:
Indentation in this file should be 2 spaces plz :)

Reply:
How come there's no tooling to autofix formatting? 😅 Can we add something?

Reply:
Hmm that's a good question! I think erblint can do this, let me look into it.

Reply:
Ok, turns out erblint doesn't have a linter for indentation. We'll have to write one ourselves ☹️

Reply:
Gotcha.. also, I typically don't have a strong opinion on code formatting (which is why I want a robot to fix it for me) _but_ the topic of spaces vs. tabs has been floating around Twitter lately. Maybe we should move to tabs? https://twitter.com/Rich_Harris/status/1541761871585464323

Reply:
Oh wow I had no idea! Unfortunately the Ruby standard is 2 spaces, so we'd be fighting an uphill battle. That said, I think a strong argument can be made for tooling making tabs vs spaces entirely transparent to the developer.